starter-template/README.md

```diff
@@ -10,97 +10,60 @@ You can then go ahead and edit the template to fit your needs.
+- **contracts** - Your custom smart contracts should be located under here. You can use any Arc contract simply by importing it. This is an example import `import "@daostack/arc/contracts/universalSchemes/UniversalScheme.sol";`.
+- **migrations** - This is a truffle migration scripts folder. You can write Truffle migration script for deploying your DAO or smart contracts and place the script under that folder. We already created a file for you there with some explanation on how to use it with Arc. The new DAO is deployed using @daostack/migration library.
+- **starter-app** - This folder is initialized with React App which uses DAOstack client library `@daostack/client` to interact with contracts and the subgraph. These will be your Dapp front-end files. In development environment we have a `webpack` server running which looks for the changes made in `starter-app/src` folder and compiles them into client side JS files. You can can find the starting file `App.js` which you can use for your project.
+- **subgraph** - This is based on graphprotocol and uses mappings defined in `src/mappings` to index the blockchain events and store them in a `postgres` database as Entities described by `graphQl` schema. The `subgraph` deploy script uses output of migrations in `data/migration.json` to get the contract addresses. You can find mappings for the base contracts here and can add more mappings and schema for your custom contracts
+-**data** - This folder contains the `testDaoSpec.json`, which is file used to specify details for the new DAO to be deployed, feel free to change it according to your project needs. It also contains an `example.env`, which has example environment variables set, copy this file to your project root i.e. `starter-template` folder as `.env` and edit the variables accordingly. Lastly, there is a `migration.json` file which will contain the output after running the migration
 
-- `dist` - Your Dapp front-end files should be placed under this folder. You can see there we already creaated a basic HTML page imorting JQuery and the compiled js file from `src` (named `main.js`).
+## Running your project on local testnet:
 
-## Running your project:
+After downloading the project and docker:
 
-Enter the project folder from the terminal and type the following:
-
-After downloading the project:
+- copy the `example.env` from `data` directory as `.env` in the project directory
+- Update the `.env` file with your `SEED_PHRASE` and <TESTNET-NAME>`_INFURA_URL`
+- Enter the project folder from the terminal (starter-template) and type the following:
 
 ```
-npm install
-npm install -g nps
-npm install -g npx
+npm run launch:docker
```

Comment:
Maybe it's too obvious but we can just remind here to run `npm install`.

Reply:
Oh yeah right now I have only added Instructions for running inside docker, In which case `npm install` is run while building docker image and not needed separately.
But I can add another section with instructions to run without docker.

Reply:
I agree, so we can just leave it this way.